<launch>

    <group ns="Casper_Sensing">
      <node pkg="casper" name="Casper_Sensing" type="Casper_Sensing.py"/>
    </group>

    <group ns="Casper_Motion">
      <node pkg="casper" name="Motion" type="Casper_Motion.py"/>
    </group>


</launch>
